src/meta/proto-conv/tests/it/user_proto_conv.rs

```diff
@@ -806,7 +809,10 @@ fn test_old_stage_file() -> anyhow::Result<()> {
             NaiveDate::from_ymd_opt(2022, 9, 16).unwrap(),
             NaiveTime::from_hms_opt(0, 1, 2).unwrap(),
         );
-        let user_id = mt::principal::UserIdentity::new("datafuselabs", "datafuselabs.rs");
```

Comment:
In order not to change the test result, hostname is left here. But do we need to construct useridentity here with a new method and change the test result at the same time cc @youngsofun @drmingdrmer